test/blackbox-tests/test-cases/coq/empty-modules.t/run.t

```diff
@@ -7,6 +7,4 @@ correctly.
   >  (modules))
   > EOF
 
-Currently raises an internal excpetion.
-  $ dune build 2>&1 | head -n 1
-  Error: exception Failure("hd")
+  $ dune build 2>&1
```

Comment:
```suggestion
The dependency detection is able to handle the empty modules case.
  $ dune build
```

Reply:
I guess this is not specific to the dep detection (could introduce a bug in other parts of the rule generation, stanza parsing, etc) right?

Reply:
Sure, maybe leave a comment like: "This builds fine as expected."

Reply:
The important thing is that we want this case to be handled well by all our logic.

Reply:
Ok, added, tho the first line of the test already had something similar.